peatio:amqp:deposit_coin: process with pid 2712 started.
I, [2018-05-02T16:30:12.428155 #2712]  INFO -- : Connected to AMQP broker (prefetch: default)
Terminating threads ..
Stopped.
peatio:amqp:deposit_coin: process with pid 15745 started.
I, [2018-05-02T17:26:41.557282 #15745]  INFO -- : Connected to AMQP broker (prefetch: default)
Terminating threads ..
Stopped.
peatio:amqp:deposit_coin: process with pid 19173 started.
I, [2018-05-02T17:39:38.027479 #19173]  INFO -- : Connected to AMQP broker (prefetch: default)
I, [2018-05-02T18:29:32.655347 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-02T18:29:32.798505 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-02T18:29:32.798843 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-02T18:29:32.799072 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"en"}
F, [2018-05-02T18:29:33.239119 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-02T18:29:33.244475 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-08T16:40:11.465178 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-08T16:40:11.655411 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-08T16:40:11.658978 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-08T16:40:11.662054 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"en"}
F, [2018-05-08T16:40:11.890969 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-08T16:40:11.895173 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-08T18:49:07.046884 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-08T18:49:07.249035 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-08T18:49:07.253871 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-08T18:49:07.254856 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"en"}
F, [2018-05-08T18:49:07.321085 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-08T18:49:07.324965 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-08T18:49:44.018586 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"zh-CN"}
F, [2018-05-08T18:49:44.106707 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-08T18:49:44.108742 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-08T18:49:44.109475 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"zh-CN"}
F, [2018-05-08T18:49:44.154334 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-08T18:49:44.156297 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-08T18:49:46.831725 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-08T18:49:46.884786 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-08T18:49:46.888556 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-08T18:49:46.888796 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"en"}
F, [2018-05-08T18:49:46.942024 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-08T18:49:46.944784 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:03:51.090809 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-10T14:03:51.555111 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:03:51.559493 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:03:51.562600 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"en"}
F, [2018-05-10T14:03:51.809539 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:03:51.810751 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:07:44.434012 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-10T14:07:44.619702 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:07:44.631171 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:07:44.639508 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"en"}
F, [2018-05-10T14:07:44.735632 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:07:44.738860 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:29:33.224144 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-10T14:29:33.382899 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:29:33.387872 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:29:33.389715 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"en"}
F, [2018-05-10T14:29:33.451327 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:29:33.453223 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:30:42.232712 #19173]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-10T14:30:42.378075 #19173] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:30:42.386156 #19173] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:30:42.433675 #19173]  INFO -- : Received: {"payment_address_id":2,"currency":"eth","locale":"en"}
F, [2018-05-10T14:30:42.496452 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:30:42.497683 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:31:41.663944 #19173]  INFO -- : Received: {"payment_address_id":3,"currency":"eth","locale":"en"}
F, [2018-05-10T14:31:41.729442 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:31:41.729711 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:32:04.699599 #19173]  INFO -- : Received: {"payment_address_id":3,"currency":"eth","locale":"en"}
F, [2018-05-10T14:32:04.747405 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:32:04.747747 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T14:34:39.992039 #19173]  INFO -- : Received: {"payment_address_id":4,"currency":"eth","locale":"en"}
F, [2018-05-10T14:34:40.068591 #19173] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T14:34:40.075001 #19173] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
Terminating threads ..
Stopped.
peatio:amqp:deposit_coin: process with pid 21397 started.
I, [2018-05-10T15:30:44.496517 #21397]  INFO -- : Connected to AMQP broker (prefetch: default)
I, [2018-05-10T15:30:44.501964 #21397]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-10T15:30:44.902154 #21397] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T15:30:44.904675 #21397] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T15:30:44.905020 #21397]  INFO -- : Received: {"payment_address_id":5,"currency":"eth","locale":"en"}
F, [2018-05-10T15:30:44.909365 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T15:30:44.909912 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T15:30:44.910064 #21397]  INFO -- : Received: {"payment_address_id":6,"currency":"eth","locale":"en"}
F, [2018-05-10T15:30:44.912935 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T15:30:44.913116 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T15:30:44.913262 #21397]  INFO -- : Received: {"payment_address_id":6,"currency":"eth","locale":"en"}
F, [2018-05-10T15:30:44.916499 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T15:30:44.916779 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T15:34:27.287014 #21397]  INFO -- : Received: {"payment_address_id":7,"currency":"eth","locale":"en"}
F, [2018-05-10T15:34:27.330232 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T15:34:27.330766 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T15:34:27.331575 #21397]  INFO -- : Received: {"payment_address_id":7,"currency":"eth","locale":"en"}
F, [2018-05-10T15:34:27.338097 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T15:34:27.338448 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T18:00:38.557642 #21397]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-10T18:00:38.998902 #21397] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:00:38.999278 #21397] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T18:00:38.999511 #21397]  INFO -- : Received: {"payment_address_id":7,"currency":"eth","locale":"en"}
F, [2018-05-10T18:00:39.011402 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:00:39.012704 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T18:01:19.351052 #21397]  INFO -- : Received: {"payment_address_id":8,"currency":"eth","locale":"en"}
F, [2018-05-10T18:01:19.363713 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:01:19.365276 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T18:01:19.365625 #21397]  INFO -- : Received: {"payment_address_id":8,"currency":"eth","locale":"en"}
F, [2018-05-10T18:01:19.372000 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:01:19.372660 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T18:02:07.305287 #21397]  INFO -- : Received: {"payment_address_id":9,"currency":"eth","locale":"en"}
F, [2018-05-10T18:02:07.316434 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:02:07.317815 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T18:02:07.318231 #21397]  INFO -- : Received: {"payment_address_id":9,"currency":"eth","locale":"en"}
F, [2018-05-10T18:02:07.326788 #21397] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:02:07.327163 #21397] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:73:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
Terminating threads ..
Stopped.
peatio:amqp:deposit_coin: process with pid 51042 started.
I, [2018-05-10T18:11:07.248756 #51042]  INFO -- : Connected to AMQP broker (prefetch: default)
I, [2018-05-10T18:11:36.202634 #51042]  INFO -- : Received: {"payment_address_id":10,"currency":"eth","locale":"en"}
F, [2018-05-10T18:11:36.412138 #51042] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:77:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:65:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:11:36.417821 #51042] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:77:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:65:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T18:11:36.418228 #51042]  INFO -- : Received: {"payment_address_id":10,"currency":"eth","locale":"en"}
F, [2018-05-10T18:11:36.425589 #51042] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:77:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:65:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:11:36.425953 #51042] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:77:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:65:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
Terminating threads ..
Stopped.
peatio:amqp:deposit_coin: process with pid 51939 started.
I, [2018-05-10T18:15:41.060230 #51939]  INFO -- : Connected to AMQP broker (prefetch: default)
I, [2018-05-10T18:16:06.857458 #51939]  INFO -- : Received: {"payment_address_id":11,"currency":"eth","locale":"en"}
-----------------------host------------
scheme
user
password
host
port
path
query
opaque
fragment
parser
Your_eth_server_ip
80
-----------------------end host------------
F, [2018-05-10T18:16:06.903849 #51939] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:16:06.904642 #51939] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T18:16:06.904870 #51939]  INFO -- : Received: {"payment_address_id":11,"currency":"eth","locale":"en"}
-----------------------host------------
scheme
user
password
host
port
path
query
opaque
fragment
parser
Your_eth_server_ip
80
-----------------------end host------------
F, [2018-05-10T18:16:06.910343 #51939] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T18:16:06.910626 #51939] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T19:19:48.809222 #51939]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-10T19:19:48.922681 #51939] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T19:19:48.923086 #51939] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-10T19:19:48.923374 #51939]  INFO -- : Received: {"payment_address_id":11,"currency":"eth","locale":"en"}
-----------------------host------------
scheme
user
password
host
port
path
query
opaque
fragment
parser
Your_eth_server_ip
80
-----------------------end host------------
F, [2018-05-10T19:19:48.929596 #51939] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-10T19:19:48.929869 #51939] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-11T12:02:47.887116 #51939]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-11T12:02:48.019625 #51939] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-11T12:02:48.023031 #51939] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-11T12:02:48.023501 #51939]  INFO -- : Received: {"payment_address_id":11,"currency":"eth","locale":"en"}
-----------------------host------------
scheme
user
password
host
port
path
query
opaque
fragment
parser
Your_eth_server_ip
80
-----------------------end host------------
F, [2018-05-11T12:02:48.417097 #51939] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-11T12:02:48.419088 #51939] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-11T12:25:00.371166 #51939]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-11T12:25:00.474828 #51939] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-11T12:25:00.476411 #51939] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-11T12:25:00.476664 #51939]  INFO -- : Received: {"payment_address_id":11,"currency":"eth","locale":"en"}
-----------------------host------------
scheme
user
password
host
port
path
query
opaque
fragment
parser
Your_eth_server_ip
80
-----------------------end host------------
F, [2018-05-11T12:25:00.485693 #51939] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-11T12:25:00.488027 #51939] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-11T12:35:53.550393 #51939]  INFO -- : Received: {"payment_address_id":1,"currency":"btc","locale":"en"}
F, [2018-05-11T12:35:53.661635 #51939] FATAL -- : CoinRPC::ConnectionRefusedError (CoinRPC::ConnectionRefusedError)
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-11T12:35:53.663183 #51939] FATAL -- : /Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:46:in `rescue in http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:39:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:32:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:14:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
I, [2018-05-11T12:35:53.663427 #51939]  INFO -- : Received: {"payment_address_id":11,"currency":"eth","locale":"en"}
-----------------------host------------
scheme
user
password
host
port
path
query
opaque
fragment
parser
Your_eth_server_ip
80
-----------------------end host------------
F, [2018-05-11T12:35:53.670544 #51939] FATAL -- : getaddrinfo: nodename nor servname provided, or not known (SocketError)
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
F, [2018-05-11T12:35:53.670794 #51939] FATAL -- : /Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `initialize'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `open'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:879:in `block in connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:878:in `connect'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rest-client-1.6.8/lib/restclient/net_http_ext.rb:51:in `request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:78:in `http_post_request'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:61:in `handle'
/Users/yyong/rbdev/peatio/app/services/coin_rpc.rb:23:in `method_missing'
/Users/yyong/rbdev/peatio/app/models/worker/deposit_coin_address.rb:12:in `process'
/Users/yyong/rbdev/peatio/lib/daemons/amqp_daemon.rb:61:in `block (2 levels) in <top (required)>'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer.rb:56:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/channel.rb:1672:in `block in handle_frameset'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `call'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:84:in `block (2 levels) in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:80:in `block in run_loop'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `catch'
/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:79:in `run_loop'
E, [2018-05-11T22:30:53.648741 #51939] ERROR -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: Exception in the reader loop: AMQ::Protocol::EmptyResponseError: Empty response received from the server.
E, [2018-05-11T22:30:53.652916 #51939] ERROR -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: Backtrace: 
E, [2018-05-11T22:30:53.656316 #51939] ERROR -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: 	/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/amq-protocol-1.9.2/lib/amq/protocol/frame.rb:60:in `decode_header'
E, [2018-05-11T22:30:53.656609 #51939] ERROR -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: 	/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:208:in `read_next_frame'
E, [2018-05-11T22:30:53.657305 #51939] ERROR -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: 	/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
E, [2018-05-11T22:30:53.658116 #51939] ERROR -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: 	/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
E, [2018-05-11T22:30:53.660476 #51939] ERROR -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: 	/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
E, [2018-05-11T22:30:53.660774 #51939] ERROR -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: 	/Users/yyong/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
W, [2018-05-11T22:30:53.666688 #51939]  WARN -- #<Bunny::Session:70312235090040 guest@127.0.0.1:5672, vhost=/>: Recovering from a network failure...
